### PR TITLE
Bundle line is_closed and name patches

### PR DESCRIPTION
**Pull request by gcandal**

### CODE REVIEW

File: src/budy/models/order.py [hunk 1]
```diff
@@ -693,6 +704,10 @@ def lines_csv_url(self, absolute = False):
             absolute = absolute
         )
 
+    @appier.operation(name = "Fix Closed Lines")
+    def fix_closed_s(self):
+        if self.is_closed(): self.close_lines_s()
```
Comment (joamag) [inline src/budy/models/order.py]: use this as pre-condition is_closed first

File: src/budy/models/bundle_line.py [hunk 2]
```diff
@@ -100,13 +106,15 @@ def order_name(self):
...
         base.BudyBase.pre_save(self)
+        if self.closed: return
         self.calculate()
         self.measure()
         self.ensure_valid()
 
     def calculate(self, currency = None, country = None, force = False):
         currency = currency or self.currency
         country = country or self.country
+        if self.total_taxes or self.total: return
```
Comment (joamag) [inline src/budy/models/bundle_line.py]: put self.is_closed() at the top of the method (pre-condition)

File: src/budy/models/bundle_line.py [hunk 3]
```diff
@@ -80,6 +81,11 @@ class BundleLine(base.BudyBase):
         type = int
     )
 
+    closed = appier.field(
```
Comment (joamag) [inline src/budy/models/bundle_line.py]: index this value

**Comment by gcandal** [comment]
@joamag done

**Pull request merged**